Library/Formula/ppl.rb

```diff
@@ -0,0 +1,25 @@
+class Ppl < Formula
+  desc "Parma Polyhedra Library"
+  homepage "http://bugseng.com/products/ppl"
+  url "ftp://ftp.cs.unipr.it/pub/ppl/releases/1.1/ppl-1.1.tar.xz"
```

Comment:
http is preferred over ftp, I'd suggest to use this `url`:
http://bugseng.com/products/ppl/download/ftp/releases/1.1/ppl-1.1.tar.xz